news/_posts/2018-11-29-native-modules-and-smart-menu.md

```diff
@@ -0,0 +1,52 @@
+---
+layout: post
+title:  "Native modules and smart menu"
+subtitle: "A bit of historical background and the refactoring plan"
+date:   2018-11-29 12:00:00
+authors: [ BilalDriche ]
+icon: icon-code
+tags:
+ - community
+ - contribution
+ - forge
+ - modules
+---
+
+As an open source project, PrestaShop is continuously being enriched by the company teams, external contributors, developers, etc. Thanks to all the community, it keeps growing in the right direction. The PrestaShop Team is proud and honored to read everybody's feedback, ideas, translations and pull requests, it helps improving the PrestaShop project and making it better for everyone!
+
+It is the strength of this community that renders each version of the software more powerful, with new features and contributions added. As of November 2018, more than 80 native modules are included in the upcoming PrestaShop 1.7.5 to fit merchants' needs, and we need your feedback!
+
+
+## A bit of historical background
+
+We all know it, when PrestaShop 1.7 was first released in 2016 it included less native modules than 1.6. At that point in time, some had been considered obsolete. Which turned out to be wrong. Many merchants claimed there was a real need for some of the removed modules regarding their business. Therefore, it appears important to bring some the most important modules back in the software… and here we are!
+We are now launching this project with you, not only bringing modules back but also aiming at improving user experience and fixing most bugs. 
+So, on the PrestaShop’s Product Management team, we have chosen the most important native modules based on community feedbacks on [github](https://github.com/PrestaShop/PrestaShop), on [uservoice](http://feedback.prestashop.com/forums/387864-prestashop-1-7-x) and on real-life [events](https://www.prestashop.com/en/events), and we defined their features based on needs, trends and legislations.
+We also want to rework some of current native modules of Prestashop 1.7.
+ 
+For the next releases, we aim to bring back Wishlist, Product review and Legal compliance modules and rework Reassurance block and VAT number module. 
+
+We try to include in every release some of your ideas and request.For example,
+stock location, not in a module but in the core, will be back soon in 1.7.5.0.
+
+
+## Smart menu
+
+For one of the first project the focus is on one of the most important PrestaShop modules, the Main menu module, which displays the Front Office menu. Many merchants use the current version, but they are not fully satisfied it can be improved and UX configuration is quite difficult for some users.
```

Comment:
répétition de "one of" non nécessaire, je dirais "for the first project, the focus is on one of the most..."

"but they are not fully satisfied it can be improved and UX configuration is quite difficult for some users." : cette phrase n'a pas de sens...